coursier.bzl

```diff
@@ -227,7 +219,7 @@ def _compute_dependency_tree_signature(artifacts):
         if artifact["file"] != None:
             artifact_group.extend([
                 artifact["sha256"],
-                artifact["file"],
+                artifact["file"].replace("\\", "/"),  # Make sure we represent files in a stable way cross-platform
```

Comment:
why not use `_normalize_to_unix_path`? here and below

Reply:
Done